cmr/cmr-workflow.cwl

```diff
@@ -14,6 +14,7 @@ inputs:
   cmr_collection : string
   cmr_start_time: string
   cmr_stop_time: string
+  limits: string?
   cmr_edl_user: string?
   cmr_edl_pass: string?
```

Comment:
i wonder if we want this to be the TOKEN provided by urs and not the actual edl username/password? sharing the token is a lot less scare than sharing the password. Functionally they are basically the same thing.

Reply:
i should state, i'm not against using this "for now" and then updating it to a EDL token in the future

Reply:
UDS provided CMR search task currently doesn't support EDL. It can in the future to support collections that are hidden from public. Data download (next task) will require EDL username/password.